pkg/scaffold/controller/controller.go

```diff
@@ -245,7 +249,7 @@ func (r *Reconcile{{ .Resource.Kind }}) Reconcile(request reconcile.Request) (re
 	found := &appsv1.Deployment{}
 	err = r.Get(context.TODO(), types.NamespacedName{Name: deploy.Name, Namespace: deploy.Namespace}, found)
 	if err != nil && errors.IsNotFound(err) {
-		log.Printf("Creating Deployment %s/%s\n", deploy.Namespace, deploy.Name)
+		log.Info("Creating Deployment %s/%s\n", deploy.Namespace, deploy.Name)
```

Comment:
logr interface is different than stdlog. I don't think this will produce what you want it to produce.

See https://github.com/kubernetes-sigs/controller-runtime/issues/187 for a similar bug.
Here's the API: https://github.com/go-logr/logr/blob/master/logr.go#L111
Example usage: https://github.com/go-logr/logr/blob/master/examples/usage_example.go#L119

Reply:
Thanks for helping me to catch this! 
And I will open a PR to fix https://github.com/kubernetes-sigs/controller-runtime/issues/187 later